doc/string/codepoints.rdoc

```diff
@@ -0,0 +1,6 @@
+Returns an array of the codepoints (integer ordinals) in +self+;
```

Comment:
I'm not sure the description "integer ordinals" explains codepoints. Maybe something like "integer value of each character in the string".

Reply:
Modified.